src/theme/MarkdownStyler.module.css

```diff
@@ -26,3 +26,11 @@
 .markdownStyler p {
   margin: 0.75em 0;
 }
+
+.markdownStyler a {
+  color: blue;
```

Comment:
Can you find out the color styling that standard Chakra links use?

Reply:
Check out theme/links.ts

Reply:
It would be ideal if we could use chakra class name here.

Reply:
I have picked the color teal.500 due to the [chakra document here ](https://v2.chakra-ui.com/docs/components/link#link-inline-with-text)